uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_email.scss

```diff
@@ -0,0 +1,28 @@
+$email-heading-line-height: 40px;
+
+.email {
+  margin: 40px 0;
```

Comment:
To be honest adding multiple properties is not a good solution, can we create a mixin which would do this for us?